packages/amplify-provider-awscloudformation/lib/system-config-manager.js

```diff
@@ -242,6 +244,16 @@ function getProfileConfig(profileName) {
   return normalizeKeys(profileConfig);
 }
 
+function loadConfigFromPath(profilePath) {
```

Comment:
This method is not needed, and should not be here, system-config-manager is only meant to  deal with system level configuration settings, such as contents in the ~/.aws/ folder and in the values in the environmental variables. 
This method is only used in the configuration-manager, in which is should be directly calling the new context.amplify.readJsonFile(.) to return aws configurations stored in the file.